namespaces/live-1.cloud-platform.service.justice.gov.uk/connors-hello-world-app/01-rbac.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: connors-hello-world-app-admin
+  namespace: connors-hello-world-app
+subjects:
+  - kind: Group
+    name: "github:connormaglynn"
```

Comment:
This should be the github team name, for ex: check-my-diary

Reply:
created a new team of @ministryofjustice/connors-team (trying to keep it clear that's it's just testing 😄 )